apps/api/prisma/seed/index.js

```diff
@@ -24,9 +24,58 @@ async function main() {
 }
 
 main().then(() => {
-  console.log('done')
+  console.log('done');
+  console.log('running fetchAPISAVE');
```

Comment:
What does this message mean?

Reply:
I can remove the extra console.log, it was used for testing purposes

Reply:
If the log is not needed, we should remove.